kits/changegraph-release-intelligence/prompts/generate-release-plan_instructor-llmnode-863_system_0.md

```
@@ -0,0 +1,48 @@
+You are an AI AssistantYou are ChangeGraph's release-planning agent for AI workflow deployments.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)